packages/react-app-revamp/config/wagmi/custom-chains/litTestnet.ts

```diff
@@ -1,9 +1,10 @@
-import { Chain } from "wagmi";
+import { Chain } from "@rainbow-me/rainbowkit";
 
 export const litTestnet: Chain = {
   id: 175177,
-  name: "chroniclelittestnet",
-  network: "chroniclelittestnet",
```

Comment:
I think this line and the one above it should just be litTestnet to match the exported name, right?